Vineyard/Vineyard/GroupListView.swift

```diff
@@ -9,7 +9,7 @@ import SwiftUI
 
 struct GroupListView: View {
     @EnvironmentObject var loginViewModel: LoginViewModel
-    @StateObject var viewModel: GroupsListViewModel
+    @State var viewModel: GroupsListViewModel
```

Comment:
This shouldn't be passed into the view. You can declare it here and remove it in `HomeView`.
```suggestion
    @State var viewModel: GroupsListViewModel = .init()
```